uint32 age
---
uint32 result